base_layer/wallet_ffi/src/lib.rs

```diff
@@ -1144,14 +1144,53 @@ pub unsafe extern "C" fn encrypted_value_as_bytes(
 /// memory leak
 #[no_mangle]
 pub unsafe extern "C" fn encrypted_value_encrypt(
```

Comment:
I don't see the need for this method? I think it should be removed from the ffi interface

Reply:
Yip, this one was undecided in my head as well when I added it, so agree it can be removed.